resources/fty-config.cfg.in

```diff
@@ -23,6 +23,8 @@ available-features
     network-host-name = /etc/hostname
     network-agent-settings = /var/lib/fty/etn-ipm2-network/etn-ipm2-network.json
     network-proxy = /etc/default/fty-proxy
+    timezone-settings = /etc/fty/fty-datetime.cfg
```

Comment:
/etc/fty/fty-timezone.cfg
 and check the update related to fty-rest (config.ecpp) and fty-core
 (must be tested runtime on a FI)